include/exec/start_now.hpp

```diff
@@ -0,0 +1,221 @@
+      __operation(_StgRef __stgref, _R&& __r) : __stgref_(__stgref), __rcvr_((_R&&)__r) {}
+
+      void join() const noexcept override {
+        stdexec::set_value(std::move(__rcvr_));
+      }
+
+      void start() noexcept {
+        const __joiner* expected = &__empty_joiner_;
+        if (!__stgref_().__joiner_.compare_exchange_strong(expected, this)) {
+          join();
+        }
+      }
+    };
+
+    template <class _StgRef>
+    struct __sender {
+      using sender_concept = stdexec::sender_t;
+      using __id = __sender;
+      using __t = __sender;
+
+      using __completions_t = stdexec::completion_signatures<stdexec::set_value_t(), stdexec::set_stopped_t()>;
+
+      _StgRef __stgref_;
+
+      using connect_t = stdexec::connect_t;
+      template <stdexec::__decays_to<__sender> _Self, stdexec::receiver _Receiver>
+        requires stdexec::receiver_of<_Receiver, __completions_t>
+      STDEXEC_MEMFN_DECL(
+        auto connect)(this _Self&& __self, _Receiver __rcvr) //
+        noexcept(std::is_nothrow_move_constructible_v<_Receiver>)
+          -> __operation<_StgRef, std::remove_cvref_t<_Receiver>> {
+        return {static_cast<_Self&&>(__self).__stgref_, static_cast<_Receiver&&>(__rcvr)};
+      }
+
+      auto get_completion_signatures(stdexec::__ignore = {}) -> __completions_t {
+        return {};
+      }
+    };
+
+    template <class _EnvId, class _AsyncScopeId, class... _SenderId>
+    struct __storage {
+      using _Env = stdexec::__t<_EnvId>;
+      using _AsyncScope = stdexec::__t<_AsyncScopeId>;
+      using __receiver_t = __receiver<__ref_t<__storage>>;
+      using __sender_t = __sender<__ref_t<const __storage>>;
+      using __env_t = stdexec::__env::__join_t<stdexec::__env::__with<stdexec::inplace_stop_token, stdexec::get_stop_token_t>, _Env>;
+      template<class _S>
+      using __nested_t = decltype(stdexec::__declval<_AsyncScope&>().nest(stdexec::__declval<_S&&>()));
+
+      mutable std::atomic<const __joiner*> __joiner_;
+      mutable std::atomic<int> __pending_;
+
+      STDEXEC_ATTRIBUTE((no_unique_address))
+      stdexec::inplace_stop_source __source_;
+      STDEXEC_ATTRIBUTE((no_unique_address))
+      __env_t __env_;
+      STDEXEC_ATTRIBUTE((no_unique_address))
+      stdexec::__decayed_tuple<stdexec::connect_result_t<__nested_t<stdexec::__t<_SenderId>>, __receiver_t>...> __op_state_;
+
+      template<class _Sender>
+      auto __construct(_Sender&& __sndr) noexcept {
+        return [&, this](){return stdexec::connect(static_cast<_Sender&&>(__sndr), __receiver_t{__ref(*this)});};
+      }
+
+      __storage(_Env __env, _AsyncScope& __scope, stdexec::__t<_SenderId>... __sndr)
+        : __joiner_(&__empty_joiner_)
+        , __pending_(0)
+        , __source_()
+        , __env_(stdexec::__env::__join(stdexec::__env::__with{__source_.get_token(), stdexec::get_stop_token}, std::move(__env)))
+        , __op_state_(stdexec::__conv{__construct(__scope.nest(__sndr))}...) {
```

Comment:
... here you don't have to use `__conv`; you can just direct-initialize the tuple with a pack of calls to `stdexec::connect`. that works because `stdexec::__tup::__tuple` is an aggregate.

when you make this change, you will need to replace calls like `__apply(fn, static_cast<Tup&&>(tup))` to `tup.apply(fn, static_cast<Tup&&>(tup))`. (`__tuple::apply` is a static member; once stdexec can assume support for deducing-this, we can replace it everywhere with `tup.apply(fn)`.)